rfc/rfc017-distributed-network-grpc-v2.md

```diff
@@ -241,15 +239,16 @@ A node MUST make sure to only add transactions of which all previous transaction
 
 The `State` message is sent as response to various conditions of the gossip protocol (see §5).
 The `State` message MUST contain a new `conversationID`.
-The `State` message contains a `XOR` value and an `LC`.
+The `State` message contains an `XOR` value and an `LC`.
 The `LC` value MUST equal the highest Lamport Clock value of all transaction references included in the `XOR` calculation.
 If no transactions are present, an all-zero `XOR` and `LC` of 0 is sent.
 
-This might look exactly like the `Gossip` message, but the type of response is different so it is a different type of message.
+This might look exactly like the `Gossip` message, but the type of response is different, so it is a different type of message.
 
 #### 6.2.2 IBLT response
 
 When a peer receives a `State` message and the `XOR` differs, it SHOULD respond with a `TransactionSet` message.
+This is a response type message, so it MUST include the sent `conversationID`.
```

Comment:
this is the same as line 256. Verification of the conversationID (with one from `State`) when receiving a `TransactionSet` is missing in §6.2.3

Reply:
added extra line for validation.